packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.js

```diff
@@ -55,7 +55,7 @@ Meteor.methods({
 		if (data.language !== 'en') {
 			const localeFn = Meteor.call('loadLocale', data.language);
 			if (localeFn) {
-				Function(localeFn)();
+				Function(moment.localeFn)();
```

Comment:
This seems like this isn't really solving the problem.  The code originally was getting the locale storing it in localeFn and trying to call localeFn as a function.  Makes sense why that might not work.

But... with this change you're simply calling a function called localeFn on moment. I don't believe this is actually a valid function.